.github/workflows/check-new-releases.yaml

```diff
@@ -0,0 +1,335 @@
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ${{ fromJson(needs.download-builds.outputs.matrix) }}
+    steps:
+      - name: Print passed vars
+        run: |
+          echo "name: ${{ inputs.name }}"
+          echo "lp-project: ${{ inputs.lp-project }}"
+          echo "lp-repo: ${{ inputs.lp-repo }}"
+          echo "lp-branch-prefix: ${{ inputs.lp-branch-prefix }}"
+          echo "tarball-regex: ${{ inputs.tarball-regex }}"
+          echo "matrix version: ${{ matrix.version }}"
+      
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Download releases
+        uses: actions/download-artifact@v3
+        with:
+          name: releases-${{ inputs.name }}
+
+      - name: Decompress all versions and list them
+        run: unzip ${{ env.OUTPUT_DIR }}-${{ inputs.name }}.zip; ls "${{ env.OUTPUT_DIR }}";
+
+      - name: Get tarball name and software version
+        id: get-tarball-name
+        run: |
+          cd output/${{ matrix.version }}/
+          count=$( ls ${{ inputs.tarball-regex }} | wc -l )
+          if [ $count != 1 ]; then
+            echo "Tarball file not found with regex: ${{ inputs.tarball-regex }}"
+            exit 1
+          fi
+          tarball_filename=$(ls ${{ inputs.tarball-regex }})
+          echo "TARBALL_FILENAME=$tarball_filename" >> $GITHUB_ENV
+          version=$( echo "${{ matrix.version }}" | awk -F '-' '{print $NF}' )
+          echo "VERSION=$version" >> $GITHUB_ENV
+          echo "Tarball filename: ${{ env.TARBALL_FILENAME}}"
+          echo "Software version: ${{ env.VERSION }}"
+      
+      - name: Create credentials file
+        env:
+          CONSUMER_KEY: ${{ secrets[inputs.lp-consumer-key] }}
+          ACCESS_TOKEN: ${{ secrets[inputs.lp-access-token] }}
+          ACCESS_SECRET: ${{ secrets[inputs.lp-access-secret] }}
+        run: |
+          cat <<EOF >> ${{ env.LP_CREDENTIALS }}
+          [1]
+          consumer_key = ${CONSUMER_KEY}
+          consumer_secret = 
+          access_token = ${ACCESS_TOKEN}
+          access_secret = ${ACCESS_SECRET}
+          EOF
+
+      - name: Install Python dependencies
+        run: pip install -r requirements.txt
+
+      - name: LP release of packages
+        id: create-new-lp-release
+        run: |
+          echo "Tarball name: ${{ env.TARBALL_FILENAME }} | version: ${{ env.VERSION }}"
+          if [[ "${{ inputs.lp-repo }}" != *"/soss/"* ]]; then
+            python3 launchpad_release.py \
+              --app "${{ inputs.name }}" \
+              --project "${{ inputs.lp-project }}" \
+              --version "${{ env.VERSION }}" \
+              --tarball "${{ env.OUTPUT_DIR }}/${{ matrix.version }}/${{ env.TARBALL_FILENAME }}" \
+              --credentials ${{ env.LP_CREDENTIALS }}
+          else
+            echo "Do not upload any artifacts for repo: ${{ inputs.lp-repo }}"
+          fi
+      - name: Delete previous release
```

Comment:
**todo, out-of-scope** Uhm, I'm not sure we should delete previous release. I mean, we should rather add new release with new tags. I have seen that we do this in LP as well. 

The idea is that when we release something new, we should probably find a sensible way to tag that, both on LP and on Github